t/op/sigsystem.t

```diff
@@ -19,10 +19,12 @@ SKIP: {
     require Time::HiRes;
 
     my @pids;
+    my $reaped_child = 0;
```

Comment:
I don't think you need this variable. Instead of testing `if $reaped_child`, you could just test `if @pids`.

Reply:
Good suggestion. Thanks!